src/brsTypes/components/RoSGNode.ts

```diff
@@ -344,6 +365,25 @@ export class RoSGNode extends BrsComponent implements BrsValue, BrsIterable {
             return child;
         },
     });
+
+    /* Returns the node that is a descendant of the nearest component ancestor of the subject node whose id field matches the given name,
+        otherwise return invalid.*/
```

Comment:
can you just add a comment that it's DFS instead of the doc's BFS?

Reply:
https://developer.roku.com/docs/references/brightscript/interfaces/ifsgnodedict.md#findnodename-as-string-as-object

> The search for the descendant node is a breadth-first search that includes child nodes in nodes that are declared as custom components defined in other XML component files

Reply:
will do. went through a bunch of scenarios with Sean yesterday and we feel pretty confident it is DFS.

Reply:
Docs wrong again, thanks for taking the time of checking this on the box